integration_tests/tests/apply/apply.cy.ts

```diff
@@ -696,8 +705,8 @@ context('Apply', () => {
 
       // Then the application should be submitted to the API
       cy.task('verifyApplicationUpdate', application.id).then(requests => {
-        expect(requests).to.have.length(36)
-        const requestBody = JSON.parse(requests[35].body)
+        expect(requests).to.have.length(37)
+        const requestBody = JSON.parse(requests[36].body)
```

Comment:
Can probably do this here:

```
const requestBody = JSON.parse(requests[requests.length - 1].body)
```

Reply:
Also maybe `applicationSubmissionBody` might be a better name, as `requestBody` is a bit generic.